contracts/cw-dao/src/contract.rs

```diff
@@ -841,6 +854,23 @@ mod tests {
             .sum()
     }
 
+    fn get_tally_by_vote(app: &App, dao_addr: &str, proposal_id: u64, vote: Vote) -> Uint128 {
```

Comment:
helper function used for testing, we were before hardcoding only getting vote_yes